src/transformers/file_utils.py

```diff
@@ -1465,8 +1465,9 @@ def is_tensor(x):
     """Tests if ``x`` is a :obj:`torch.Tensor`, :obj:`tf.Tensor` or :obj:`np.ndarray`."""
     if is_torch_available():
         import torch
+        import torch.fx
```

Comment:
this one is a bit tricky since it's a recent addition in pytorch, so it can only be loaded if the pytorch version is right. 

Do you know if we want pt-1.8.0 or higher? I think it should be 1.8.0 - we can adjust later if need be.

and of course it'd impact `if isinstance` below - so probably split `isinstance` in 2 parts and condition `if isinstance(x, torch.fx.Proxy)` check on pytorch version 

typically we do it by implementing `is_torch_fx_available` - see a whole bunch of those in `file_utils.py`

So 
```
if is_torch_fx_available():
    import torch.fx
....

if  is_torch_fx_available() and isinstance(x, torch.fx.Proxy):
    return True
```

and version you get from:

```
if version.parse(torch.__version__) >= version.parse("1.8"): 
```
so now you can implement `is_torch_fx_available` in `file_utiles.py` and import it here.